spec/coverage_helper.rb

```diff
@@ -1,19 +1,21 @@
 # frozen_string_literal: true
 
-require 'simplecov'
+unless !!ENV['CI']
+  require 'simplecov'
```

Comment:
I also find it annoying in dev when running single tests. I'm wondering if we should run this only when a certain variable is given.

Reply:
If you think so, we could introduce `COVERAGE=true` to get the report. That's what I did the other time I introduced Simplecov in a project.

Reply:
That would be great. Maybe in another PR?

Reply:
yes. Feel free :joy:

Reply:
Done: #7078